core/src/main/java/org/opensearch/sql/data/model/ExprDateValue.java

```diff
@@ -28,14 +29,15 @@
 @RequiredArgsConstructor
 public class ExprDateValue extends AbstractExprValue {
 
+  //private static final DateTimeFormatter DATE_FORMATTER_VARIABLE_NANOS_OPTIONAL = ;
```

Comment:
```suggestion
```

Reply:
Resolved in https://github.com/Bit-Quill/opensearch-project-sql/pull/134/commits/2016222f524af6c2edd56afa940a53181efd3b1c.